### ISSUE TITLE
Install guide

### ISSUE THREAD

**Issue by langovoi** [hexchan/hexchan-engine#1]
https://hexchan.org looks awesome!

Can you provide install guide or docker-compose example?

**Comment by binakot** [comment]
@hexchan :octocat:

**Comment by hexchan** [comment]
@langovoi @binakot You're right, there are no install docs yet. I'm suprised somebody is interested. I'll make it a first priority task then!

**Comment by binakot** [comment]
Waiting for it ☕️ 
When it will be done, I update my Docker env.

**Comment by langovoi** [comment]
@hexchan any updates?

**Comment by hexchan** [comment]
@langovoi @binakot I've updated installation guide, see INSTALL.md. You can use it to deploy project locally, but you've got to make some extra steps to use it in production mode. I'll write them down a little bit later.

**Comment by binakot** [comment]
> @langovoi @binakot I've updated installation guide, see INSTALL.md. You can use it to deploy project locally, but you've got to make some extra steps to use it in production mode. I'll write them down a little bit later.

Awesome! Waiting for prod guide. Then I will update my PR with docker env deploy. And after that I will try to deploy your hexchan in our production 😎

**Comment by binakot** [comment]
@hexchan any updated with prod env?

**Comment by binakot** [comment]
Docker-compose environment is ready to use. Now I'm waiting for your review, @hexchan. Than I will add docker-stack (swarm) environment. You should just install Docker and type in terminal from project folder:

```bash
docker-compose build
docker-compose up -d
docker exec -it hexchan_app python src/manage.py createsuperuser
```

And check that everything works like a charm :)

P.S. still waiting for prod env recommendations :octocat:

**Comment by binakot** [comment]
I updated my PR with docker-stack. Now it can be run in docker swarm cluster, but I have a problem with uwsgi configuration. I see TODO in the section in INSTALL.md file. Waiting for it to make my stack works.

**Comment by binakot** [comment]
@hexchan, hello! Any updates for wsgi section in install guide? At this moment I have a problem with this step. I updated `uwsgi.ini` file https://github.com/hexchan/hexchan-engine/pull/2/files#diff-733321f33a9f128ade1a7bdec67f878d, but it doesn't work :( Please, help! :octocat:

**Comment by binakot** [comment]
Okay, the job is done! Now we have a docker environment to run this imageboard without install guide :)
